docs/blog/setup.md

```diff
@@ -0,0 +1,41 @@
+The Sysl open source setup
+======================
+
+We have recently set up what we think of as good engineering practises for our enterprise-backed open source project [Sysl](https://github.com/anz-bank/sysl). The following is a record of systems and practises we consider essential and the products and platforms we have chosen (in parenthesis):
```

Comment:
s/think of as/consider to be/

Reply:
Done, but then we've got a word repetition in the first 2 sentences. So changed the second one to deem which os a strange word, isn't it?